src/chains/ethereum/transaction/src/transaction-factory.ts

```diff
@@ -193,19 +200,57 @@ export class TransactionFactory {
     switch (txType) {
       case TransactionType.EIP1559AccessList:
         return EIP1559FeeMarketTransaction.fromTxData(
-          txData.slice(1) as EIP1559FeeMarketDatabasePayload,
+          txData.slice(1) as EIP1559FeeMarketRawTransaction,
```

Comment:
This diff looks weird, but all that really happened here is that I added a new function, `fromTypeAndTxData` (below), and some type names have changed.